server/src/routes/admin.ts

```diff
@@ -18,14 +18,14 @@ adminRouter.post("/signup", async (req: Request, res: Response) => {
       await prisma.$disconnect();
       return res.status(403).json({ message: "Admin email already exists" });
     }
-
     const hashedPassword: string = await bcrypt.hash(password, 8);
     await prisma.admin.create({
       data: {
         email,
         hashedPassword: hashedPassword,
       },
-    });
+    })
```

Comment:
What is the point of removing that semicolon?

Reply:
Actually nothing, I was just seeing and trying something...

I didn't do it intentionally tbh. :)

Reply:
Be serious man we are building an industry level project not a kidde project.
I might sound harsh but I am just giving you honest feedback hope you understand that part. I can merge anything and everything but if I do that you will not groww as a developer.